src/main/java/nl/knaw/dans/ingest/core/service/mapper/DepositToDvDatasetMetadataMapper.java

```diff
@@ -107,6 +110,7 @@ public class DepositToDvDatasetMetadataMapper {
     }
 
     public Dataset toDataverseDataset(
+        Deposit deposit,
```

Comment:
At least five more arguments are taken unconditionally from the deposit. Why not reduce the number of arguments?

Reply:
Actually, I think we should not pass the complete deposit to this function as this will lead to further mixing of concerns.